packages/application-shell/src/configure-store.ts

```diff
@@ -39,23 +71,22 @@ const sdkMiddleware = createSdkMiddleware({
   getTeamId: selectTeamIdFromLocalStorage,
 });
 
-export const applyDefaultMiddlewares = (...middlewares) =>
+export const applyDefaultMiddlewares = (...middlewares: Middleware[]) =>
   applyMiddleware(...middlewares, thunk, loggerMiddleware);
 
 // We use a factory as it's more practicable for tests
 // The application can import the configured store (the default export)
 export const createReduxStore = (
-  preloadedState = { requestsInFlight: null },
+  preloadedState = { requestsInFlight: undefined },
   // additional middleware, used for testing
-  additionalMiddlewares = []
+  additionalMiddlewares: Middleware[] = []
 ) => {
   const store = createStore(
     createInternalReducer(),
     preloadedState,
     compose(
       applyDefaultMiddlewares(
         ...additionalMiddlewares,
-        sentryTrackingMiddleware,
```

Comment:
This middleware wasn't necessary anymore as:
- in the global error handler we are already sending the error to sentry
- in the handleActionError we also send the error to sentry now

I found that having the middleware caused some unnecessary indirection of the logic. It should be easier to follow along now.